<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Style</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>66</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Family</string>
    </property>
   </widget>
   <widget class="AlexLineEdit" name="lineEditFamily">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>471</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxBold">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Bold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxItalic">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Italic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxUnderline">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>95</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Underline</string>
    </property>
   </widget>
   <widget class="AlexLineEdit" name="lineEditSize">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>230</y>
      <width>501</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="Alexlabel" name="labelfox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>471</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>the quick brown fox likes cse 335</string>
    </property>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>451</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AlexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>alexlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Alexlabel</class>
   <extends>QLabel</extends>
   <header>alexlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
